mongodbatlas/resource_mongodbatlas_project_test.go

```diff
@@ -346,37 +345,6 @@ func testAccCheckMongoDBAtlasProjectExists(resourceName string, project *matlas.
 	}
 }
 
-func TestAccProjectRSProject_CreateWithAdvancedCluster(t *testing.T) {
```

Comment:
Shouldn't we still run this in case change_detection happens only for mongodbatlas_project? Or are you saying the test itself isn't relevant for project resource?

Reply:
> Or are you saying the test itself isn't relevant for project resource

Yes, this is irrelevant to the project resource itself and we already run it as part of all the advanced tests. We always create a project first and then create the advanced cluster in those tests